docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -22,8 +22,9 @@ Windows.
   https://software.intel.com/sites/default/files/managed/0f/c8/Intel-SGX-PSW-Release-Notes-for-Windows-OS.pdf)
 - [Microsoft Visual Studio 2017](https://visualstudio.microsoft.com/vs/older-downloads/)
 - [Git for Windows 64-bit](https://git-scm.com/download/win)
-- [OCaml for Windows 64-bit](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/)
 - [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe)
+- [OCaml for Windows 64-bit](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/)
+- Openssl for Windows - Add mingw64/bin to your path
```

Comment:
Unnecessary addition here, we explicitly depend on the Git installation of openssl and we don't want to confuse a dev who might end up with a separate independent install of OpenSSL.

Reply:
This does not get added by default and I have to always add this so that openssl gets picked up. We do depend on openssl for Windows to generate certificates, etc. 

I can remove this comment. But we still need a fix. An elegant way would be to establish the path to openssl at the start like we do for BASH with OPENSSL. Will file an issue or make the change with this PR.

Reply:
This does not get added by default and I have to always add this so that openssl gets picked up. We do depend on openssl for Windows to generate certificates, etc.

I can remove this comment. But we still need a fix. An elegant way would be to establish the path to openssl and set OPENSSL at the start like we do for BASH. Will file an issue or make the change with this PR.

Reply:
References to OpenSSL have been removed at this time.

Reply:
Removed this line.